### ISSUE TITLE
Doesn't detect any power values

### ISSUE THREAD

**Issue by ke-bo80** [Breina/PowerTagGateway#16]
I'm using Panel Server Universal 600. With Power Tag flex, everything works fine, and I can transfer it to HA, but it doesn't detect any power values. It only shows 2 alarm sensors, nothing else comes over. Any ideas why?

![IMG_5141](https://github.com/Breina/PowerTagGateway/assets/48104694/4ce46533-5c67-4312-b7eb-3453901b0d2c)

**Comment by Breina** [comment]
Is it possible that you configured it as a Link Gateway instead of Panel Server during the setup?

**Comment by ke-bo80** [comment]
Everything is possible, but I've deleted everything and tried again. However, I don't get prompted at any point to choose between Link Gateway or Panel Server. It automatically detects my panel server. But got the same result, no values."![image](https://github.com/Breina/PowerTagGateway/assets/48104694/59a61cfb-d94b-4c67-b9cb-347d0a26a693)

**Comment by Breina** [comment]
Can you please check if the logs say anything?

Also which version are on currently on?

**Comment by ke-bo80** [comment]
I'm on EcoStruxure PowerTag Link Gateway v. 0.2.1. 
Home Assistant Core 2024.3.0 - 
Supervisor 2024.02.1 - 
Operating system 12.0.

It seems to be the same issue as Issue #15, I just have a Panel Server 600. 
I have 2 errors in my log. Perhaps they mean something to you?

Logger: homeassistant.components.sensor
Kilde: helpers/entity_platform.py:350
integration: Sensor (dokumentation, problemer)
Første forekomst: 7. marts 2024 kl. 22.08.47 (1 forekomster)
Senest logget: 7. marts 2024 kl. 22.08.47

Error while setting up powertag_gateway platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 350, in _async_setup_platform
    await asyncio.shield(awaitable)
  File "/config/custom_components/powertag_gateway/sensor.py", line 62, in async_setup_entry
    PowerTagPowerFactor(client, modbus_address, tag_device),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/powertag_gateway/sensor.py", line 606, in __init__
    if is_r(product_type):
       ^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/powertag_gateway/entity_base.py", line 111, in is_r
    feature_class = [fc for fc in FeatureClass if product_type in fc.value][0]
                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^

IndexError: list index out of rangeLogger: homeassistant.components.button
Kilde: helpers/entity_platform.py:350
integration: Knap (dokumentation, problemer)
Første forekomst: 7. marts 2024 kl. 22.08.47 (1 forekomster)
Senest logget: 7. marts 2024 kl. 22.08.47

Error while setting up powertag_gateway platform for button
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 350, in _async_setup_platform
    await asyncio.shield(awaitable)
  File "/config/custom_components/powertag_gateway/button.py", line 55, in async_setup_entry
    class_r = is_r(product_type)
              ^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/powertag_gateway/entity_base.py", line 111, in is_r
    feature_class = [fc for fc in FeatureClass if product_type in fc.value][0]
                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range

**Comment by Breina** [comment]
Can you please upgrade to v0.2.2-alpha12 (you'll need to [enable beta](https://hacs.xyz/docs/faq/beta/)) and check the logs? I'm particularly interested in the logs right before it crashes.

**Comment by ke-bo80** [comment]
I have now updated to 0.2.2-alpha12 and it has found the panel server, but now there is no power tag. Here is a log error.

Logger: root
Kilde: custom_components/powertag_gateway/schneider_modbus.py:554
integration: PowerTag Link Gateway (dokumentation, problemer)
Første forekomst: 14.30.47 (3 forekomster)
Senest logget: 14.30.47

Unknown product type: 17210

**Comment by Breina** [comment]
Ah I see. That's an undocumented product type code.

![image](https://github.com/Breina/PowerTagGateway/assets/3009914/932d3fce-a45d-494b-98c6-48f86ade5c95)

Can you tell me the model / commercial reference of that wireless device please?

**Comment by ke-bo80** [comment]
it is A9MEM1560 and A9MEM1570

**Comment by Breina** [comment]
Yeah that was stupid of me. Sorry about that!

![image](https://github.com/Breina/PowerTagGateway/assets/3009914/23093a9c-8046-4092-a897-a91d7a56cd23)

Fixed in 0.2.2. Please verify.

**Comment by ke-bo80** [comment]
It works. 
Thank you so much for your help, it's really awesome
